src/lib/Scenes/Show/Show.tsx

```diff
@@ -0,0 +1,61 @@
+import { Theme } from "@artsy/palette"
+import React from "react"
+import { NavigatorIOS, ViewProperties } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { DetailContainer as DetailScreen } from "./Screens/Detail"
+import { MoreInfoContainer as MoreInfoScreen } from "./Screens/MoreInfo"
+
+import { Show_show } from "__generated__/Show_show.graphql"
+
+interface Props extends ViewProperties {
+  show: Show_show
+}
+
+export class Show extends React.Component<Props> {
+  navigator?: NavigatorIOS
+
+  handleMoreInformationPressed = () => {
+    if (!this.navigator) {
+      throw new Error("navigator is undefined")
+    }
+
+    this.navigator.push({
```

Comment:
We could avoid holding a ref and passing this method down as a prop by doing the navigation directly from within `Detail`, but getting Relay wired up seemed awkward to me - `Detail` would have to include the `MoreInfo_show` fragment. I'm open to changing this though.

Reply:
Na i think it's ok to do it that way